RecoTracker/FinalTrackSelectors/plugins/TrackLwtnnClassifier.cc

```diff
@@ -0,0 +1,88 @@
+#include "RecoTracker/FinalTrackSelectors/interface/TrackMVAClassifier.h"
+
+#include "FWCore/Framework/interface/EventSetup.h"
+#include "FWCore/Framework/interface/ESHandle.h"
+
+#include "DataFormats/TrackReco/interface/Track.h"
+#include "DataFormats/VertexReco/interface/Vertex.h"
+
+#include "getBestVertex.h"
+
+//from lwtnn
+#include "TrackingTools/Records/interface/TrackingComponentsRecord.h"
+#include "lwtnn/LightweightNeuralNetwork.hh"
+
+namespace {
+  struct lwtnn {
+    lwtnn(const edm::ParameterSet& cfg):
+      lwtnnLabel_(cfg.getParameter<std::string>("lwtnnLabel"))
+    {}
+
+    static const char *name() { return "TrackLwtnnClassifier"; }
+
+    static void fillDescriptions(edm::ParameterSetDescription& desc) {
+      desc.add<std::string>("lwtnnLabel", "trackSelectionLwtnn");
+    }
+
+    void beginStream() {}
+    void initEvent(const edm::EventSetup& es) {
+      edm::ESHandle<lwt::LightweightNeuralNetwork> lwtnnHandle;
+      es.get<TrackingComponentsRecord>().get(lwtnnLabel_, lwtnnHandle);
+      neuralNetwork_ = lwtnnHandle.product();
+    }
+
+    float operator()(reco::Track const & trk,
+                     reco::BeamSpot const & beamSpot,
+                     reco::VertexCollection const & vertices,
+                     lwt::ValueMap & inputs) const {
+      // lwt::ValueMap is typedef for std::map<std::string, double>
+      //
+      // It is cached per event to avoid constructing the map for each
+      // track while keeping the operator() interface const.
+
+      Point bestVertex = getBestVertex(trk,vertices);
+
+      inputs["trk_pt"] = trk.pt();
+      inputs["trk_eta"] = trk.eta();
+      inputs["trk_lambda"] = trk.lambda();
+      inputs["trk_dxy"] = trk.dxy(beamSpot.position()); // is the training with abs() or not?
+      inputs["trk_dz"] = trk.dz(beamSpot.position()); // is the training with abs() or not?
+      inputs["trk_dxyClosestPV"] = trk.dxy(bestVertex); // is the training with abs() or not?
+      inputs["trk_dzClosestPV"] = trk.dz(bestVertex); // is the training with abs() or not?
```

Comment:
the "questionable" comments are still here. 
https://github.com/cms-sw/cmssw/pull/21717#discussion_r158309983 implied that these will be modified to be simple statements.

Reply:
I now pushed a commit modifying the comments.